openff/units/units.py

```diff
@@ -76,7 +76,7 @@ class UnitRegistry(pint.UnitRegistry):
     _measurement_class = Measurement
 
 
-DEFAULT_UNIT_REGISTRY = UnitRegistry(get_defaults_path())
+DEFAULT_UNIT_REGISTRY = UnitRegistry(get_defaults_path(), cache_folder=":auto:")
```

Comment:
(not blocking) Fascinating, and good find. Looks like this landed in the [Pint 0.19 line](https://pint.readthedocs.io/en/0.19.2/performance.html?highlight=cache#speed-up-registry-instantiation), around 2022.